tests/nvidiagpu/gpu_suite_test.go

```diff
@@ -26,12 +27,31 @@ func TestGPUDeploy(t *testing.T) {
 	RunSpecs(t, "GPU", Label(tsparams.Labels...), reporterConfig)
 }
 
+type mustGatherConfig struct {
+	envVar     string
+	reportPath string
+}
+
 var _ = JustAfterEach(func() {
 	specReport := CurrentSpecReport()
 	reporter.ReportIfFailed(
 		specReport, currentFile, tsparams.ReporterNamespacesToDump, tsparams.ReporterCRDsToDump, clients.SetScheme)
-	artifactDir := inittools.GeneralConfig.GetReportPath("gpu-must-gather")
-	if err := reporter.RunMustGather(artifactDir, 5*time.Minute); err != nil {
-		glog.Errorf("Failed to collect must-gather for the GPU operator, %v", err)
+
+	mustGathers := []mustGatherConfig{
```

Comment:
Same problem. You will have to add a new variable for every must gather. But you already have a must-gather script, so make it collect whatever information is needed.